docs/docs/_index.md

```diff
@@ -8,7 +8,7 @@ type: docs
 [![Discord](https://img.shields.io/discord/697882427875393627?style=flat-square)](https://discord.gg/KExRgMb)
 [![Github](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RedisTimeSeries/RedisTimeSeries/)
 
-RedisTimeSeries is a Redis module that adds a time series data structure to Redis.
+Time Series is a Redis module that adds a time series data structure to Redis.
```

Comment:
We cannot say that Time Series is a module.

Reply:
@dwdougherty we'll probably have to do the same change to all the other features.